### ISSUE TITLE
Upon passing at the beginning of the game, you automatically win

### ISSUE THREAD

**Issue by Kai-Richardson** [RJMNillo/CS301ASSN5GoTest#9]
Description:
For either of the AI players, if you pass as your first move, you automatically win the game.

Reproduction Steps:
Pass as your first move

This is due to this snippet of code:
https://github.com/RJMNillo/CS301ASSN5GoTest/blob/a452190150ac5d56edddbb146cb750fce313ef5b/app/src/main/java/com/example/cs301assn5go/Go/GoLocalGame.java#L53-L63

You don't check if both scores are 0, and therefore this bug occurs.

**Comment by braedenlane** [comment]
We have now fixed this according to the games rules. If player 0 were to pass on the first turn with an empty board, it would make sense for the second player to also want to pass, since when scores are calculated, it looks for area surrounded. Since the board is empty, each player's score would be 0, but since player 2 is awarded a +.5 bonus (komi) for going second, they would win. So since, we've added komi to the scoring, it would make sense for the game to end after the player passes on the first turn, as the second player would win because of the scoring algorithm.
